src/EA.Weee.Core/AatfEvidence/EvidenceNoteData.cs

```diff
@@ -6,19 +6,43 @@
     using System.Linq;
     using System.Text;
     using System.Threading.Tasks;
+    using AatfReturn;
+    using Prsd.Core;
 
     public class EvidenceNoteData
     {
         //TODO: Other evidence note properties required
         public int Reference { get; set; }
```

Comment:
Can we return the Note Id primary key on this (and map it) we will have to use it to generate the View / Edit links